environment.yml

```diff
@@ -1,38 +1,53 @@
+# 1. Install a conda distribution.
+#    https://mamba.readthedocs.io/en/latest/installation/mamba-installation.html
+# 2. Run: `mamba env create -f environment.yml`
+# 3. Activate: `conda activate spyglass`
+#
+# (lines intentionally left blank)
+#
+#
 name: spyglass
 channels:
   - conda-forge
   - defaults
+  # - pytorch # dlc-only
   - franklab
   - edeno
 dependencies:
-  - python>=3.9,<3.10
+  - bottleneck
+  # - cudatoolkit=11.3 # dlc-only
+  # - cupy # dlc-only
+  # - dask-cuda # dlc-only
+  # - ffmpeg # dlc-only
+  - ipympl
   - jupyterlab>=3.*
-  - pydotplus
-  - dask
+  # - libgcc # dlc-only
+  - matplotlib
+  - non_local_detector
+  - numpy<1.24
   - pip
   - position_tools
-  - numpy<1.24
-  - track_linearization>=2.3
+  - pybind11  # req by mountainsort4 -> isosplit5
+  - pydotplus
+  - pyfftw<=0.12.0  # ghostipy req. install from conda-forge for Mac ARM
+  - python>=3.9,<3.10
+  - pytorch<1.12.0
+  - replay_trajectory_classification
   - ripple_detection
-  - non_local_detector
-  - matplotlib
   - seaborn
-  - bottleneck
-  - ipympl
-  - tqdm
```

Comment:
Why is the removal of this required by datajoint? I also rely on this in the `non_local_detector` package so I don't know if this is completely going away.

Reply:
Maybe miscommunication on my part, maybe this was marked resolved as to say 'nvm' - but I removed `tqdm` and `pymysql` to cut down on conda resolution time. Because DJ requires them, we don't need to specify here